### PR TITLE
tips(serenity-js): added 3-work-with-frames

### PR DESCRIPTION
**Pull request by jan-molak**
## 🌟 Type of Pull Request (check all that apply):

- [x] New Selenium Tip
- [ ] Reporting a Bug
- [ ] New Feature
- [ ] Documentation Changes
- [ ] Other _(please replace this text with details)_

## 🚀 Description

Per my conversation with @diemol, this is the first Serenity/JS tutorial demonstrating how to use Serenity/JS to interact with The Internet.

## 🤝 Related Issue

-  This PR is related to Issue #

## ✅ Checklist

Please check off the following items as applicable:
- [x] Code builds and runs locally without errors.
- [x] Content has been checked for spelling and grammar errors. 
- [x] Follows the [CODE OF CONDUCT](https://github.com/saucelabs/elemental-next/blob/main/CODE_OF_CONDUCT.md).
- [x] Follows the [CONTRIBUTING](https://github.com/saucelabs/elemental-next/blob/main/CONTRIBUTING.md) guidelines.

## 🔎 Testing

- [x] This has been tested
- [ ] This has not been tested _(please replace this text explaining why)_
- [ ] I need help testing this
- [ ] Testing will be done via another Issue

## 📝 Additional Notes

>### 🎉 Inclusive Content Reminder
>
>We truly value diversity and strive to create a welcoming and inclusive community. Please review our [CODE OF CONDUCT](https://github.com/saucelabs/elemental-next/blob/main/CODE_OF_CONDUCT.md) and [CONTRIBUTING](https://github.com/saucelabs/elemental-next/blob/main/CONTRIBUTING.md) guidelines to make doubly sure your contribution meets and upholds our principles and values.

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -18,15 +18,16 @@
     "@docsearch/react": "3.6.0",
     "@docusaurus/core": "^3.1.1",
     "@docusaurus/preset-classic": "^3.1.1",
+    "@docusaurus/module-type-aliases": "^3.1.1",
```
Comment (diemol) [inline frontend/package.json:21]: Why do we need changes to the main `package.json`

**Review by diemol — CHANGES_REQUESTED** [review]
I think the overall looks good, just a couple of comments that will help us get this one correctly and pave the way for the next ones!

File: frontend/package.json (hunk 1 above)
Comment (jan-molak) [reply frontend/package.json:21]: It looks like all the Docusaurus-releated packages are marked as `dependencies`, and [only `@docusarus/module-type-aliases` is marked as a `devDependency`](https://github.com/saucelabs/elemental-next/blob/4b55d9599ff64c3eb448ff6b5da8d4cbf0a77aae/frontend/package.json#L17-L30). It didn't seem intentional, so I thought I'd clean that up while I was in the area.

If you prefer to keep it as it was I can revert that?

File: .github/workflows/javascript-tips.yml [hunk 2]
```diff
@@ -52,6 +52,10 @@ jobs:
           timeout_minutes: 20
           max_attempts: 3
           command: |
+            # Install Playwright browsers
+            cd frontend
+            npx --no playwright install --with-deps
+
```
[the remark this answers is not in the document]
Comment (jan-molak) [reply .github/workflows/javascript-tips.yml:58]: Sure, let me add that.

File: frontend/package.json (hunk 1 above)
Comment (jan-molak) [reply frontend/package.json]: I reverted the change

File: .github/workflows/javascript-tips.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (jan-molak) [reply .github/workflows/javascript-tips.yml]: Done

**Comment by jan-molak** [comment]
Outstanding tasks:
- [ ] Remove callouts to specific line numbers to follow the new convention, in the style of [Tip 5](https://deploy-preview-671--elementalselenium.netlify.app/tips/5-select-from-a-dropdown)
- [ ] Add a high-level description of Serenity/JS, in the style of [Tip 71](https://elementalselenium.com/tips/71-mobile-testing-pyramid)
  - [ ] Move the links to Serenity/JS API docs and handbook from Tip 3

**Pull request merged**